test/Interop/Cxx/templates/Inputs/dependent-types.h

```diff
@@ -26,20 +26,20 @@ template<class T, class U>
 M<U> differentDependentArgAndRet(M<T> a) { return {a.value}; }
 
 template<class T>
-M<T> dependantReturnTypeInffered(T a) { return {a}; }
```

Comment:
Sorry to nit pick, but in the future (you don't have to do it this time), please make the rename a separate NFC. This way, if you other change gets reverted, we can keep the rename. Also, it makes the rest of the code easier to review, because I can focus on what actually changed. 

Thanks for making the codebase better, though!

Reply:
Cool cool, sounds good!